docs/troubleshooting/offline-queue-trigger.md

```diff
@@ -0,0 +1,117 @@
+# Offline queue doesn't trigger on network status change
```

Comment:
General housekeeping for this PR to align it with the rest of the docs/codebase.

- Can we remove all the typescript decoration and prefer javascript
- Remove angular decorations, prefer basic redux setup please
- Remove any middleware that isn't redux/redux-offline specific (e.g. redux-observable)
- Please make note that this is if you're using `createOffline` and want to compose yourself not the default setup.

I think that only a working example and highlighting the change as you have done is needed. Should aim to keep it short and sweet.

Reply:
I feel like we can include the redux-observable setup in a separate chapter called `Redux-Observable` inside the recipes section. And BTW thanks for contributing! 🎉